Setting-up-Rhetos-for-HTTPS.md

```diff
@@ -0,0 +1,54 @@
+# Setting up Rhetos for HTTPS
+
+In order for Rhetos to work on HTTPS you need to add a new endpoint in web.config file.
+
+Insert following code:

```

Comment:
This sentence was probably originally written for an old version of Rhetos.
[Web.config](https://github.com/Rhetos/Rhetos/blob/master/Source/Rhetos/Web.config) already contains the `security` elements by default, it should be _updated_, not added.

Reply:
I'm not sure I understand this. I changed it to "In order for Rhetos to work on HTTPS you need to update security elements in web.config file." Hope this is ok.